tests/cassettes/test_address_bytestring.yaml

```diff
@@ -1,17 +1,15 @@
 interactions:
 - request:
-    body: !!python/unicode address%5Bstate%5D=DELEGACI%C3%B3N+BENITO+JU%C3%A1REZ
+    body: !!python/unicode '{"address": {"state": "DELEGACI\u00f3N BENITO JU\u00e1REZ"}}'
```

Comment:
Is this the correct behavior we are expecting here? We have a test `test_address_bytestring` to ensure these get created correctly, were we wanting it to converto to the actual letters or be as they show here?

Reply:
Bytestrings are getting passed through _utf8 as they were before in the urlencoder. 

```
>>> print(easypost._utf8(b'DELEGACI\xc3\xb3N BENITO JU\xc3\xa1REZ'))
DELEGACIóN BENITO JUáREZ

>>> print('DELEGACI\u00f3N BENITO JU\u00e1REZ')
DELEGACIóN BENITO JUáREZ
>>> 
```